Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -7,7 +7,7 @@
 .. _kubeproxy-free:
 
 *****************************
-Kubernetes without kube-proxy
+Kubernetes Without kube-proxy
```

Comment:
Maybe you have looked, but can we configure our spellcheck(?) to catch wrong capitalization in titles?

Reply:
Good idea! I've had a look but unfortunately they don't seem to have anything for title case. There are a couple options to treat CamelCase and fully-capitalized words differently, but nothing for title case :cry: 

Our documentation could definitely do with some love in that regard.